packages/x-data-grid/src/hooks/utils/useTimeout.ts

```diff
@@ -1 +1,48 @@
-export { default as useTimeout } from '@mui/utils/useTimeout';
+'use client';
+/**
+ * TODO, remove this file, have dependents import from:
+ * import useTimeout from '@mui/utils/useTimeout';
+ * directly.
+ */
```

Comment:
Only brought it back so we can quickly learn if this is correct or wrong (meaning so we can merge https://github.com/mui/mui-x/pull/16031, before https://github.com/mui/material-ui/pull/44897, so we test it with a smaller userbase.